pythonforandroid/build.py

```diff
@@ -305,13 +306,7 @@ def prepare_build_environment(self,
 
         check_ndk_api(ndk_api, self.android_api)
 
-        virtualenv = None
-        if virtualenv is None:
-            virtualenv = sh.which('virtualenv2')
-        if virtualenv is None:
-            virtualenv = sh.which('virtualenv-2.7')
-        if virtualenv is None:
-            virtualenv = sh.which('virtualenv')
+        virtualenv = get_virtualenv_executable()
```

Comment:
I like that you moved this somewhere else :+1:

Reply:
thanks :heart: I'm resolving this conversation so we get a better view of what's still left to do